cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts

```diff
@@ -46,7 +47,7 @@ let businessServicesList: Array<BusinessServices> = [];
 describe(["@tier3"], "Applications interlinked to tags and business service", () => {
     before("Login and Create Test Data", function () {
         login();
-
+        AssessmentQuestionnaire.enable("Legacy Pathfinder");
```

Comment:
I guess Legacy pathfinder will be used at many places we could add this in constant file